### PR TITLE
Fix cloudfront deploy docker image

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,3 @@
-FROM node:12.18.3-alpine AS base
```
Comment (Nas84) [inline Dockerfile:1]: why removing cdk cli ?

File: cloudfront-dist/deploy.sh [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-# Require TARGET_ENVIRONMENT=test 
-cd server && npm run gen-lambda && cd .. && \
-    rm ./static/index.html && \
-    # Deploy, move to deployement dir in order to have access to the app in cdk.json 
-    cd cloudfront-dist/deployment && \
-    echo "run cdk deploy --require-approval never $STACK_PREFIX_NAME-$FRONT_APP_NAME" && \
-    cdk deploy --require-approval never $STACK_PREFIX_NAME-$FRONT_APP_NAME
\ No newline at end of file
+# Require TARGET_ENVIRONMENT=test
+cd server
```
Comment (Nas84) [inline cloudfront-dist/deploy.sh:5]: i prefer the model line, if any command fails it stops

**Review by Nas84 — CHANGES_REQUESTED** [review]

File: cloudfront-dist/deploy.sh (hunk 2 above)
Comment (yvon-dblg) [reply cloudfront-dist/deploy.sh:5]: The `set -eu` instruction do the job.

File: Dockerfile (hunk 1 above)
Comment (yvon-dblg) [reply Dockerfile:1]: We use the one in cloudfront-dist directly to avoid having to set the version in two different file.

**Review by ydrisR — APPROVED** [review]
LGTM

**Review by Nas84 — APPROVED** [review]
LGTM

**Pull request merged**